Documentation/install/upgrade.rst

```diff
@@ -68,10 +90,19 @@ Micro versions within a particular minor version, e.g. 1.2.x -> 1.2.y, are
 always 100% compatible for both up- and downgrades. Upgrading or downgrading is
 as simple as changing the image tag version in the `DaemonSet` file:
 
-.. code-block:: shell-session
+.. tabs::
+  .. group-tab:: kubectl
+
+    .. parsed-literal::
 
-    kubectl -n kube-system set image daemonset/cilium cilium-agent=docker.io/cilium/cilium:vX.Y.Z
-    kubectl -n kube-system rollout status daemonset/cilium
+      kubectl -n kube-system set image daemonset/cilium cilium-agent=docker.io/cilium/cilium:vX.Y.Z
+      kubectl -n kube-system rollout status daemonset/cilium
+
+  .. group-tab:: Helm
+
+    .. parsed-literal::
+
+      helm upgrade cilium cilium/cilium --version X.Y.Z --namespace=kube-system
```

Comment:
The `helm upgrade` for micro releases here is actually much more invasive than the corresponding image tag set. `helm` needs all of the correct arguments from install to avoid overwriting the existing configurations. I wonder if we should highlight this here somehow?

Reply:
`helm upgrade` help string has this:

```
If no chart value arguments are provided on the command line, any existing customized values are carried
forward. If you want to revert to just the values provided in the chart, use the '--reset-values' flag.
```

It seems it shouldn't be necessary.

Reply:
Hmm, my `helm3` help string doesn't contain this.

I see these:

```
      --reset-values             when upgrading, reset the values to the ones built into the chart
      --reuse-values             when upgrading, reuse the last release's values and merge in any overrides from the command line via --set and -f. If '--reset-values' is specified, this is ignored
```

No indication of defaults though.